translation.rst

```diff
@@ -59,13 +59,13 @@ locale to lookup and return translated messages. Before using it, enable the
 
     .. code-block:: yaml
 
-        # app/config/config.yml
+        # config/packages/translation.yaml
         framework:
             translator: { fallbacks: [en] }
```

Comment:
`translation` != `translator` Inconsistent with the rest of the cases. RFC https://github.com/symfony/recipes/issues/252